src/lib/y2storage/align_policy.rb

```diff
@@ -7,6 +7,6 @@ module Y2Storage
   class AlignPolicy
     include StorageEnumWrapper
 
-    wrap_enum "Storage::AlignPolicy", names: [:align_end, :keep_end, :keep_size]
+    wrap_enum "AlignPolicy"
```

Comment:
great, it looks nicer now. The last idea. Maybe StorageEnum can be base class for such kind of classes? so it can be 
```
class AlignPolicy < StorageEnum
   wrap_enum "AlignPolicy"
end
```